components/avatar.css

```diff
@@ -0,0 +1,87 @@
+/* Avatar starts here */
+
+/* Avatar sizes */
+
+/* avatar xs */
+
+.avatar-xs {
+  position: relative;
+  vertical-align: middle;
+  font-size: 4.4px;
+}
+.avatar-xs img {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
```

Comment:
a suggestion, you are using a border-radius property everywhere, you can also create a separate class specifically for this.

Reply:
Yes